catalog/app/website/components/BucketGrid/BucketGrid.js

```diff
@@ -97,43 +75,99 @@ const useStyles = M.makeStyles((t) => ({
   },
 }))
 
+function Bucket({ bucket, onTagClick, tagIsMatching }) {
+  const b = bucket
```

Comment:
we should either put this into function definition/destructuring, e.g. `function Bucket({ bucket: b, onTagClick, tagIsMatching }) {` or just use `bucket` in function body